test/index.js

```diff
@@ -10,39 +10,39 @@ describe('normalize', function() {
     var type = 'string';
     var value = 'test string';
     var result = normalize(type, value);
-    expect(result).toEqual(value);
+    expect(result).toBe(value);
```

Comment:
I prefer equals always but wasn't going to nitpick (this is from your changeset).